src/libsync/propagateuploadv1.cpp

```diff
@@ -45,7 +46,7 @@ void PropagateUploadFileV1::doStartUpload()
     if (progressInfo._valid && Utility::qDateTimeToTime_t(progressInfo._modtime) == _item->_modtime ) {
         _startChunk = progressInfo._chunk;
         _transferId = progressInfo._transferid;
-        qDebug() << Q_FUNC_INFO << _item->_file << ": Resuming from chunk " << _startChunk;
+        qCDebug(lcPropagateUpload) << _item->_file << ": Resuming from chunk " << _startChunk;
```

Comment:
Hmmmm it's not really an error condition but resuming uploads is something crucial..